### PR TITLE
feat(console): add paginations for applications and resources

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

* add paginations for applications and resources
* show paginations only if the total data count is bigger than the page size

## Linear Issue Reference

None.

## Testing

Tested in the admin console.

### CODE REVIEW

File: packages/console/src/pages/ApiResources/index.tsx [hunk 1]
```diff
@@ -115,6 +121,15 @@ const ApiResources = () => {
           </tbody>
         </table>
       </div>
+      <div className={styles.pagination}>
+        {totalCount !== undefined && totalCount > pageSize && (
```
Comment (IceHe) [inline packages/console/src/pages/ApiResources/index.tsx:125]: `!!totalCount && totalCount > pageSize` simplier?

File: packages/console/src/pages/Applications/index.tsx [hunk 2]
```diff
@@ -118,6 +124,15 @@ const Applications = () => {
           </tbody>
         </table>
       </div>
+      <div className={styles.pagination}>
+        {totalCount !== undefined && totalCount > pageSize && (
```
Comment (IceHe) [inline packages/console/src/pages/Applications/index.tsx:128]: ditto

File: packages/console/src/pages/ApiResources/index.tsx [hunk 3]
```diff
@@ -26,12 +26,19 @@ import * as styles from './index.module.scss';
...
 const buildDetailsLink = (id: string) => `/api-resources/${id}`;
 
+const pageSize = 20;
+
 const ApiResources = () => {
   const [isCreateFormOpen, setIsCreateFormOpen] = useState(false);
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
-  const { data, error, mutate } = useSWR<Resource[], RequestError>('/api/resources');
+
+  const [pageIndex, setPageIndex] = useState(1);
```
Comment (wangsijie) [inline packages/console/src/pages/ApiResources/index.tsx:35]: Considering rebase #493 and remove `useState`.

**Review by wangsijie — CHANGES_REQUESTED** [review]

**Comment by wangsijie** [comment]
And I think we may need to show pagination even if total page count is 1. Better notice our designer before merging.

**Review by simeng-li — APPROVED** [review]
overall LGTM please make sure resolve the conversations before merge

**Review by IceHe — APPROVED** [review]
Overall LGTM

**Review by wangsijie — APPROVED** [review]

**Pull request merged**